fuel-client/src/client/schema/contract.rs

```diff
@@ -0,0 +1,43 @@
+use crate::client::schema::{schema, ContractId};
+
+#[derive(cynic::FragmentArguments, Debug)]
+pub struct ContractByIdArgs {
+    pub id: ContractId,
+}
+
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(
+    schema_path = "./assets/schema.sdl",
+    graphql_type = "Query",
+    argument_struct = "ContractByIdArgs"
+)]
+pub struct ContractByIdQuery {
+    #[arguments(id = &args.id)]
+    pub contract: Option<Contract>,
+}
+
+#[derive(cynic::QueryFragment, Debug)]
+#[cynic(schema_path = "./assets/schema.sdl")]
+pub struct Contract {
+    pub id: ContractId,
+}
```

Comment:
[nit] also include the bytecode, otherwise, this is no different than the ContractIdFragment type.

```suggestion
    pub bytecode: HexString,
}
```

Reply:
I left `bytecode` out intentionally because I'm try to avoid making methods like `client.contract(id)` too fat, which of course made both fragments the same.

ATM the `Contract` fragment is basically scaffolding for when we add stuff like `created_block_id`, `last_touched_block_id`, `utxo_id`, etc.